app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authorize_user, except: [:create, :new, :edit, :update]
+
+  def index
+    @users = User.order("email ASC").page(params[:page])
```

Comment:
it seems a bit weird/unintuitive that you would order by email. What was the reasoning for that?